src/jagadeeshMJun20/PreDefinedString.java

```diff
@@ -0,0 +1,46 @@
+/*1) Count Digits,Letters, UpperCase characters, LowerCase characters, Special characters from predefined string.
+
+Input: 1rRpd3F9#K(E
+Output : 
+Total digit: 3
+Total Letters: 7
+Total Uppercase: 4
+Total Lowercase: 3
+Total Special char: 2 */
+
+package jagadeeshMJun20;
+
+public class PreDefinedString {
+	void displayAllChars(String input) {
+		int digitCount=0,lowerCount=0,upperCount=0,specialCount=0,letterCount=0;
+		for(int index=0;index<input.length();index++) {
+			char ch = input.charAt(index);
+		if(	Character.isDigit(ch)) 
+			digitCount++;
+		else if(Character.isAlphabetic(ch)) {
+			letterCount++;
+			if(Character.isLowerCase(ch))
```

Comment:
Without using letterCount variable and else if(Character.isAlphabetic(ch)) block you are able to achieve the expected output.